scripts/basic-workflow-tests.sh

```diff
@@ -257,12 +258,22 @@ test_convert() {
 }
 
 test_evaluate() {
+    local MODEL_PATH="${DATA_HOME}/instructlab/models/instructlab/granite-7b-lab"
+    local BASE_MODEL_PATH="${DATA_HOME}/instructlab/models/instructlab/merlinite-7b-lab"
     task Evaluate the model
 
     if [ "$EVAL" -eq 1 ]; then
       export INSTRUCTLAB_EVAL_MMLU_MIN_TASKS=true
       export HF_DATASETS_TRUST_REMOTE_CODE=true
-      ilab model evaluate --model "${DATA_HOME}/instructlab/models/instructlab/granite-7b-lab" --benchmark mmlu
+      ilab model evaluate --model "${MODEL_PATH}" --benchmark mmlu
+      ilab model evaluate --model "${MODEL_PATH}" --benchmark mmlu_branch --sdg-path tests/testdata/mmlu_branch/ --base-model "${BASE_MODEL_PATH}"
```

Comment:
Does it make sense to be using `merlinite` as a base model to compare with `granite`? Or are we just looking to see if the workflow executes correctly here

Reply:
Right now we just want to see if the workflow executes correctly here.

@alinaryan I think we should leave a comment saying that just so that no one else reading the script wonders the same thing Nathan is.

Nathan, we've opened up https://github.com/instructlab/instructlab/issues/1642 to track integrating the eval commands into the entire ilab workflow. We'll start to work on that as we're not dependent on the other components being finished up.

Reply:
Added a comment